src/hermes/config.py

```diff
@@ -68,6 +70,12 @@ def configure(config_path: pathlib.Path):
     if 'hermes' in _config and _config['hermes']:
         return
 
+    # Load sane default paths for log files before potentially overwritting via configuration
+    _config['logging']['handlers']['logfile']['filename'] = \
+        working_path / HermesContext.hermes_cache_name / "hermes.log"
+    _config['logging']['handlers']['auditfile']['filename'] = \
+        working_path / HermesContext.hermes_cache_name / "audit.log"
+
```

Comment:
**nag:** I don't like this very much, however I have not better idea at all how to improve this right now.

Might be subject to a discussion for alpha: How should our configuration system work (especially in combination with plugins...)

Reply:
Maybe. As you say, let's discuss this later.